src/Microsoft.AspNetCore.Authentication/ClaimsTransformationContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.Security.Claims;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    public class ClaimsTransformationContext
+    {
+        public HttpContext Context { get; set; }
+        public ClaimsPrincipal Principal { get; set; }
```

Comment:
@HaoK, you offered to use only getters. Should I leave only getters and take both params in a constructor or setters are ok?